### PR TITLE
Fix answer for guess question (critical showstopper)

### PR DESCRIPTION
**Pull request by swaldmann**
1 Gb/Byte = 1 Gigabit/Byte = 1000000000 Bit/1 Byte = 1000000000 Bit/8 Bit = 125000000

[Außer 8 Bit != 1 Byte](https://en.wikipedia.org/wiki/Byte#Common_uses) 👀

### CODE REVIEW

**Review by GitP4tt — CHANGES_REQUESTED** [review]
please change the question so that the answer fits, 1 gigabyte === 1 000 000 000 bytes

**Comment by swaldmann** [comment]
Too EZ

File: public/cards/guesses.js [hunk 1]
```diff
@@ -574,7 +574,7 @@ module.exports = [
 		ranking: [],
 	}, {
 		category: "guess",
-		question: "Wieviel Bytes hat 1Gb?",
+		question: "Wieviel Bytes hat 1GB?",
```
Comment (GitP4tt) [inline public/cards/guesses.js:577]:
```suggestion
		question: "Wieviel Bytes hat 1 Gigabyte?",
```
please be more specific here, people will claim the abbreviation confused the shit out of the

**Review by GitP4tt — CHANGES_REQUESTED** [review]

File: public/cards/guesses.js (hunk 1 above)
Comment (swaldmann) [reply public/cards/guesses.js]: Trivia knowledge is underrated

**Review by GitP4tt — APPROVED** [review]

**Comment by GitP4tt** [comment]
THX for the fix. Much appreciated ❤️

**Pull request merged**